.github/workflows/claude.yml

```
@@ -36,28 +39,28 @@ jobs:
           # This is an optional setting that allows Claude to read CI results on PRs
           additional_permissions: |
             actions: read
-
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)